frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -274,6 +269,11 @@ export const insightLogic = kea<insightLogicType>({
             await breakpoint(100)
             actions.updateInsight({ tags: values.insight.tags?.filter((_tag) => _tag !== tag) })
         },
+        saveAsNewInsight: async () => {
+            const newInsight = await api.create('api/insight', values.insight)
+            toast(`New insight ${newInsight.name || newInsight.id} saved!`)
+            router.actions.push(`/i/${newInsight.short_id}`)
```

Comment:
Doing this will add multiple routes to the router and then mess up the back button.

Reply:
I forgot why I had this but it's not being used so I removed it for now